ros/api/v1/hosts.py

```diff
@@ -206,10 +186,15 @@ def build_system_filters():
         if operating_systems := request.args.getlist('os'):
             modified_operating_systems = []
             for os in operating_systems:
-                os = os.split(" ")[1]
-                if os not in OS_VERSIONS.keys():
-                    abort(400, message='Not a valid RHEL version')
-                modified_operating_systems.append(OS_VERSIONS[os])
+                # FIX: Remove below if block once we get os values as numeric from frontend
```

Comment:
Nitpick - it would be good to use existing code tags i.e FIXME.

```suggestion
               # FIXME: remove `if` block after getting OS values as numberic by frontend
```